docs/5.0.yaml

```diff
@@ -56,6 +56,9 @@ nav:
         - GCP: gcp-guide.md
         - IBM: ibm-cloud-guide.md
         - OpenSSH Guide: openssh-teleport.md
+    - Preview:
+        - Database Access: preview/database-access.md
+        - Cloud: preview/cloud.md
```

Comment:
I would rename this to `teleport-cloud.md` for great SEO points.

Reply:
The final URL will be https://goteleport.com/teleport/docs/preview/database-access, so having one more teleport would make it 3 teleport mentions in one URL, which is a bit too much